code/modules/mob/living/carbon/human/species.dm

```diff
@@ -56,6 +55,7 @@
 
 	// species flags. these can be found in flags.dm
 	var/list/species_traits = list()
+	var/list/species_blacklist = list()
```

Comment:
why this be a list and not a flag, u only got 4 things yo

Reply:
we paying init overhead for all these lists, every single one on every single species datum

Reply:
you didn't even make a lazy init list shame on you